packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -188,11 +177,22 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
           <Stack horizontal grow>
             <Stack.Item grow style={callCompositeContainerFlex()}>
               <Stack.Item styles={callGalleryStyles} grow>
-                {props.onRenderGalleryContent && (
-                  <Stack verticalFill styles={mediaGalleryContainerStyles}>
-                    {props.onRenderGalleryContent()}
-                  </Stack>
-                )}
+                <Stack verticalFill styles={mediaGalleryContainerStyles}>
```

Comment:
How are these changes affecting the mobile experience? are we supposed to see the error bars when the mobile panes are open as well?

Reply:
<img width="510" alt="Screenshot 2023-03-30 at 1 40 34 PM" src="https://user-images.githubusercontent.com/96077406/228959132-13eca59c-d427-47f8-a4ad-d34962d74a5d.png">
<img width="570" alt="Screenshot 2023-03-30 at 1 40 27 PM" src="https://user-images.githubusercontent.com/96077406/228959142-e69a7212-0612-432b-936a-ecc6fbb98925.png">